### PR TITLE
Generated main.py doesnt use resources other than wappsto/__init__.py

### PR DESCRIPTION
**Pull request by bubriks**
[#106](https://github.com/Wappsto/wappsto-python/issues/106) and [#8](https://github.com/Wappsto/wappsto-python/issues/8)

**main.py following code:**
status_service = service.get_status()
status_service.set_callback(status_callback)

service.get_network().set_callback(network_callback)

value_to_test = None
for device in service.instance.device_list:
    for value in device.value_list:
        value.set_callback(value_callback)
        value_to_test = value

**can be replaced with:**

service.set_status_callback(status_callback)
service.set_network_callback(network_callback)
service.set_value_callback(value_callback)

### CODE REVIEW

**Comment by bubriks** [comment]
service.set_status_callback(status_callback)
service.set_network_callback(network_callback)
service.set_value_callback(value_callback)
"""
status_service = service.get_status()
status_service.set_callback(status_callback)

service.get_network().set_callback(network_callback)

value_to_test = None
for device in service.instance.device_list:
    for value in device.value_list:
        value.set_callback(value_callback)
        value_to_test = value
"""

**Comment by bubriks** [comment]
simplified main.py file:
[main.txt](https://github.com/Wappsto/wappsto-python/files/4343575/main.txt)

File: wappsto/__init__.py [hunk 1]
```diff
@@ -236,3 +283,4 @@ def stop(self, save=True):
         if save:
             self.object_saver.save_instance(self.instance)
         self.wapp_log.info("Exiting...")
+        sys.exit(self.exit_code)
```
Comment (AndreasBomholtz) [inline wappsto/__init__.py]: You should never call exit in a lib!

**Comment by bubriks** [comment]
latest main.py file (contains control over sys.exit)
[main.txt](https://github.com/Wappsto/wappsto-python/files/4368231/main.txt)

**Pull request merged**